src/main/org/deidentifier/arx/criteria/EDDifferentialPrivacy.java

```diff
@@ -46,67 +46,109 @@
 public class EDDifferentialPrivacy extends ImplicitPrivacyCriterion {
     
     /** SVUID */
-    private static final long        serialVersionUID = 242579895476272606L;
+    private static final long              serialVersionUID = 242579895476272606L;
 
     /** Parameter */
-    private final double             epsilon;
+    private final double                   epsilonAnon;
     /** Parameter */
-    private final double             delta;
+    private final double                   epsilonSearch;
     /** Parameter */
-    private final int                k;
+    private final int                      steps;
     /** Parameter */
-    private final double             beta;
+    private final double                   delta;
     /** Parameter */
-    private DataSubset               subset;
+    private final int                      k;
     /** Parameter */
-    private transient DataManager    manager;
+    private final double                   beta;
     /** Parameter */
-    private transient boolean        deterministic    = false;
+    private DataSubset                     subset;
     /** Parameter */
-    private DataGeneralizationScheme generalization;

```

Comment:
This class *must* be extended in a way that is 100% backwards compatible to instances that have been serialized into project files using previous versions of ARX. This is not the case here. Please (a) do not change the order of object members, (b) do not remove members, (c) when adding new members, make sure that your code can handle deseralized classes that did not have these members set.

It is probably much easier to leave the current class EDDifferentialPrivacy as is and create a new class that extends this base class...